regression/ansi-c/null_pointer_constant1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE test-c++-front-end
```

Comment:
`test-c++-front-end`? Is this correct? 

The additions for supporting the operator seem to only be added to the `ansi-c/c_typecheck_expr.cpp`.

Reply:
Yes, this test input is also valid C++ code, so it should pass when running the same test via the C++ front-end. (And the C++ front-end falls back to the C type checker in places.)